### PR TITLE
Command handler

### PR DESCRIPTION
**Pull request by Agerr**

### CODE REVIEW

File: src/events/messageCreate.js [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+
+        if (message.author.bot || !message.content.startsWith(prefix)) return
+
+        const args = message.content.slice(prefix.length).split(/ +/)
+        const commandName = args.shift().toLowerCase()
+        const command = client.commands.get(commandName) || client.commands.find(cmd => cmd.aliases && cmd.aliases.includes(commandName))
+
+        if (!command) return
+
+        // Check if command is guid only
```
Comment (LunarianLime) [inline src/events/messageCreate.js]: Misspelled comment here (guid -> guild)

Comment (Agerr) [reply src/events/messageCreate.js]: fixed

**Review by LunarianLime — APPROVED** [review]
ice

**Comment by Agerr** [comment]
> ice

ice ice baby

**Pull request merged**